crates/bevy_solari/src/lib.rs

```diff
@@ -50,3 +74,26 @@ impl SolariPlugin {
             | WgpuFeatures::PARTIALLY_BOUND_BINDING_ARRAY
     }
 }
+
+#[derive(SystemSet, PartialEq, Eq, Debug, Clone, Hash)]
+pub struct SolariSystems;
+
+/// A resource to track whether the renderer has the required features for Solari systems.
+#[derive(Resource)]
+struct HasSolariRequiredFeatures;
+
+/// Check for the Solari required features once in startup, and insert a resource if the features
+/// are enabled.
+///
+/// Now systems can do a cheap check for if the resource exists.
+fn check_solari_has_required_features(mut commands: Commands, render_device: Res<RenderDevice>) {
```

Comment:
I'm a bit confused where is this system actually added to RenderStartup?

Reply:
I totally goofed haha. I added the system now!